examples/cra-kitchen-sink/src/image-storyshots.test.js

```diff
@@ -0,0 +1,31 @@
+import initStoryshots, { imageSnapshot } from '@storybook/addon-storyshots';
+import path from 'path';
+import fs from 'fs';
+
+// We do screenshots against the static build of the storybook.
+// For this test to be meaningful, you must build the static version of the storybook *before* running this test suite.
+const pathToStorybookStatic = path.join(__dirname, '..', 'storybook-static');
+
+const shouldRun = process.env.RUN_IMAGE_SNAPSHOTS;
```

Comment:
Is it the recommended way to switch between screenshots and regular testing? Should we document it then?

Reply:
Well I ran into some issues where I'm not easily able to run unit tests and not storyshots with image snapshots tests. That's why I added this flag to control the behavior. Using CRA (thus react-scripts) doest not allow overriding the ignore path (https://facebook.github.io/jest/docs/en/configuration.html#testpathignorepatterns-array-string) :( 

Only in this example using CRA I needed to separate image snapshots with storyshots and classic HTML snapshots with storyshots.

What do you think ? I may add some doc on the cra-kitchen-sink README to explain why we need this flag ?

Reply:
Does it mean that people that use CRA for their projects will face the same issue?

Reply:
No, they will run the tests, and if they decide to use storyshots with image snapshots, then they will build the static version right before running these tests. I documented it inside the CRA example project so they can have a decent explaination :)

Reply:
Static version is needed only for image testing, so it would be really nonconvenient. So we need to provide a good way to separate screenshots from other tests, and document it in addon's readme

Reply:
Maybe replacing `.test.js` extension with just `.js` for image tests would be a better approach that env variables? We pass the particular filename anyway

Reply:
If we do this, we won't be able to use react-scripts behavior. We then should use Jest directly & the custom Jest configuration (that's already be done by jest-addon test)

Reply:
> If we do this, we won't be able to use react-scripts behavior

Why do we need it? We don't use it for regular tests (the configuration is on root level)

Reply:
That's right but is the right thing to do when using CRA I believe :) It's supposed to be a zero-config tool :)
If that's ok with you I will set it up appart then, it will simplify the whole thing. Will let you know when ready. Thanks!

Reply:
In my view, addon-storyshots implies that you have some kind of jest setup which you have control over

Reply:
The thing is, we actually need some configuration to make imsge testing work separately. And I don't think that setting this env variable check is any simpler than adding a minimal jest config